src/translate.js

```diff
@@ -1263,6 +1263,26 @@ var multiLangData = {
         "ja": "サポアビ",
         "zh": "Support Ability",
     },
+    "supplemental_other": {
+        "en": "",
+        "ja": "",
+        "zh": "",
+    },
+    "supplemental_third_hit": {
+        "en": " (Applies to third hit)",
+        "ja": " (3回目の攻撃に)",
+        "zh": " (Applies to third hit)",
+    },
+    "supplemental_on_critical": {
+        "en": " (Applies to critical hit)",
+        "ja": " (クリティカル攻撃に)",
+        "zh": " (Applies to critical hit)",
+    },
+    "supplemental_hp_based": {
```

Comment:
if need threshold in display text,
make a skill->skill name function
and place-holder for the threshold info.

```json
"en": "Applies when remainHP is {threshold}%+", 
```

```js
case "hp_based":
     return intl.translate("supplemental_hp_based", locale).replace("{threshold}", supplemental.threshold);
```

Reply:
```json
"ja": "残りHPが {threshold} 以上の時",
```

Reply:
point of this is just to specify to what the damage is applied to. that's just too much details (check the wiki, or 'save' tab, _which probably need to be changed at some point_).